examples/no-op-match/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+# BANNER
+#banner.charset=UTF-8 # Banner file encoding.
+#banner.location=classpath:banner.txt # Banner file location.
+
+# APPLICATION SETTINGS (SpringApplication)
+#spring.main.banner-mode=console # Mode used to display the banner when the application runs.
+
+# build fails if mainClass not set in build.gradle
+# camel-spring-boot wants it set here  when main app extends FatJarRouter
+#  - see http://camel.apache.org/spring-boot.html
+spring.main.sources=org.mitre.ptmatchadapter.PtmatchAdapter
+
+# Spring Boot enables a JMX listener by default
+camel.springboot.jmxEnabled = false
+
+phHost=gatekeeper.mitre.org
```

Comment:
If you’re referring to the proxy, I don’t think that ended up being needed (hence the non-standard name).

From: Andy Gregorowicz [mailto:notifications@github.com]
Sent: Wednesday, March 2, 2016 3:55 PM
To: mitre/ptmatchadapter <ptmatchadapter@noreply.github.com>
Cc: Los, Michael E. <mel@mitre.org>
Subject: Re: [ptmatchadapter] Initial commit of patient match adapter classes w/ no-op example (#1)


In examples/no-op-match/src/main/resources/application.properties<https://github.com/mitre/ptmatchadapter/pull/1#discussion_r54789608>:

> +# BANNER

> +#banner.charset=UTF-8 # Banner file encoding.

> +#banner.location=classpath:banner.txt # Banner file location.

> +

> +# APPLICATION SETTINGS (SpringApplication)

> +#spring.main.banner-mode=console # Mode used to display the banner when the application runs.

> +

> +# build fails if mainClass not set in build.gradle

> +# camel-spring-boot wants it set here  when main app extends FatJarRouter

> +#  - see http://camel.apache.org/spring-boot.html

> +spring.main.sources=org.mitre.ptmatchadapter.PtmatchAdapter

> +

> +# Spring Boot enables a JMX listener by default

> +camel.springboot.jmxEnabled = false

> +

> +phHost=gatekeeper.mitre.org

Is there any way we could pull this out of the application?

—
Reply to this email directly or view it on GitHub<https://github.com/mitre/ptmatchadapter/pull/1/files#r54789608>.